app/uk/gov/hmrc/emailverification/services/AuditService.scala

```diff
@@ -30,38 +31,193 @@ class AuditService @Inject() (
     auditConnector: AuditConnector
 )(implicit ec: ExecutionContext) extends BackendHeaderCarrierProvider {
 
-  def sendPinViaEmailEvent(emailAddress: String, passcode: String, serviceName: String, responseCode: Int)(implicit request: Request[_]) = {
+  def sendPasscodeViaEmailEvent(emailAddress: String, passcode: String, serviceName: String, responseCode: Int)(implicit request: Request[_]) = {
```

Comment:
we need unit tests for this lot to ensure that e.g. the tags are correct, as per our agreement with TXM/whatever they call themselves these days

Reply:
👍